pkg/labels/labels.go

```diff
@@ -98,6 +98,8 @@ const (
 	// Boolean value which can be parsed with strconv.ParseBool() is required.
 	// (like "nerdctl/default-network=true" or "nerdctl/default-network=false")
 	NerdctlDefaultNetwork = Prefix + "default-network"
+
+	IP6Address = Prefix + "ip6"
```

Comment:
This line should come after the “IPAddress” line

Reply:
Make sense